frontend/src/pages/booking/apis/getMentoringDetail.ts

```diff
@@ -0,0 +1,17 @@
+import { apiClient } from '../../../common/apis/apiClient';
+
+export interface MentoringDetail {
+  id: 1;
+  mentorName: string;
+  categories: string[];
+  price: number;
+  career: number;
+  imageUrl: string;
+  introduction: string;
+}
+
+export const getMentoringDetail = async (
+  mentoringId: string,
+): Promise<MentoringDetail> => {
+  return await apiClient.get({ endpoint: `/mentorings/${mentoringId}` });
```

Comment:
p4: apiClient의 get에 제네릭을 전달해주면 반환 타입도 잘 잡히는 것 같아 👍
```suggestion
) => {
  return await apiClient.get<MentoringDetail>({ endpoint: `/mentorings/${mentoringId}` });
```

---

p4: `/mentorings`에 아까 만든 constants/API_URL 상수를 사용해봐도 좋을 것 같아 👍